packages/react-components/src/InputRpc/CennznetJsonRpc.ts

```diff
@@ -1,27 +1,25 @@
 import jsonrpc from '@polkadot/types/interfaces/jsonrpc';
-import {DefinitionRpcSub, DefinitionRpcParam, DefinitionTypeType} from '@cennznet/types';
+import {DefinitionRpcSub } from '@cennznet/types';
 import {DefinitionRpcExt} from '@polkadot/types/types';
-import cennznetBare from '@cennznet/api/rpc';
+import * as definitions from '@cennznet/types/interfaces/definitions';
```

Comment:
can you explain this code alittle more?
why 'cennzx' ? + I think it's still 'cennzxSpot' in v36

Reply:
rpc is cennzx even in rv36...
so.. If you check this code
https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/jsonrpc.ts
here they have used an example type to be used for example babe (consider type as babe).. we don't have babe, we have cennzx, genericAsset for which RPC calls are available.. I have tested ga RPC calls too.. They are working fine..

Reply:
I see, normally good to leave a comment on something unusual like this to save the next person some time understanding why it's like that